src/main/java/gregtech/common/gui/widget/orefilter/ItemOreFilterTestSlot.java

```diff
@@ -0,0 +1,106 @@
+package gregtech.common.gui.widget.orefilter;
+
+import com.google.common.collect.Lists;
+import gregtech.api.gui.IRenderContext;
+import gregtech.api.gui.ingredient.IGhostIngredientTarget;
+import gregtech.api.unification.OreDictUnifier;
+import gregtech.api.util.Position;
+import mezz.jei.api.gui.IGhostIngredientHandler;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.client.renderer.RenderItem;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.awt.*;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+public class ItemOreFilterTestSlot extends gregtech.common.gui.widget.orefilter.OreFilterTestSlot implements IGhostIngredientTarget {
```

Comment:
Can `gregtech.common.gui.widget.orefilter.OreFilterTestSlot` be imported and shortened?